cypress/fixtures/users.json

```diff
@@ -12,5 +12,12 @@
         "email": "user-1@sample.mattermost.com",
         "firstName": "Victor",
         "lastName": "Welch"
+    },
+    "user-2": {
+        "username": "samuel.tucker",
+        "password": "user-2",
+        "email": "user-2@sample.mattermost.com",
+        "firstName": "Samuel",
+        "lastName": "Tucker"
```

Comment:
The change to the account setting for user-1 was impacting `account_settings/display/channel_display_mode_spec.js`.

In another comment you mention reverting the setting, which we can do, but if this test fails in a manner that makes it so we can't revert the setting then tests that run after this one may fail due to the setting not getting reverted.

I've usually handled this by either using distinct users for each test (or creating users for each test, during execution). Or setting the account settings to a default state in a before/beforeEach block in other tests (hopefully via API or something, so that we can avoid having to drive the UI each time). 

Let me know what you want for now and I can get it done either way.

Reply:
Not sure how the change on `channel_display_mode_spec.js` is affecting this test but definitely, the change on that spec should be reverted as well.  Nice catch!

I totally agree on "test failed and setting not getting reverted" but we might end up on "stop on first fail" so that test will not continue on fail.

Use of distinct users is an easy pick but I prefer the reset of account settings via API.  I have a newly created ticket to address that - [MM-14391](https://mattermost.atlassian.net/browse/MM-14391).  I also have another ticket to add command in changing specific account setting - [MM-14390](https://mattermost.atlassian.net/browse/MM-14390).

At the moment, let's stick with "user-1" with this flow - change account setting via UI -> do the test on subject (e.g. post message) -> revert the change on account setting via UI.  My goal is to observe and address the Cypress limitation in driving the UI on this early stage of UI automation, and understand what it takes to be flaky.

Reply:
It had to do with how, when compact, the width of the post changes and that test happened to be checking specific sizes.

All changes should be pushed up now, including the latest ESLint fix.